src/services/formatting/formatting.ts

```diff
@@ -714,6 +714,11 @@ namespace ts.formatting {
 
                 processNode(child, childContextNode, childStartLine, undecoratedChildStartLine, childIndentation.indentation, childIndentation.delta);
 
+                if (child.kind === SyntaxKind.JsxText) {
+                    const range = { pos: child.getStart(), end: child.getEnd() };

```

Comment:
Object literal has no contextual type, this can lead to wierd errors. Use `const range: TextRange` or just include it inline.